common/tor/BUILD.gn

```diff
@@ -13,12 +11,20 @@ source_set("tor") {
   ]
 
   deps = [
+    ":pref_names",
     "//base",
     "//chrome/common:constants",
     "//url:url",
   ]
 }
 
+source_set("pref_names") {
```

Comment:
Pulling it out because we'll hit below assertion if we add brave/common/tor as dep in core/browser/BUILD.gn directly.
ERROR at //build/config/BUILDCONFIG.gn:658:5: assert_no_deps failed.
    target(_target_type, target_name) {
    ^----------------------------------
//components:components_browsertests has an assert_no_deps entry:
  //chrome/*
which fails for the dependency path:
  //components:components_browsertests ->
  //components/autofill/content/browser:browser ->
  //components/autofill/core/browser:browser ->
  //components/sync:sync ->
  //components/signin/core/browser:browser ->
  //components/content_settings/core/browser:browser ->
  //brave/components/content_settings/core/browser:browser ->
  //brave/common/tor:tor ->
  //chrome/common:constants